test/oatpp-swagger/test-controllers/TestController.hpp

```diff
@@ -32,9 +32,16 @@ class UserDto : public oatpp::DTO {
   DTO_FIELD(Int32, id);
   DTO_FIELD(String, firstName, "first-name");
   DTO_FIELD(String, lastName, "last-name");
+  DTO_FIELD_INFO(referral) {
+    info->pattern = "^[^\\s]+$";
+  };
   DTO_FIELD(String, referral, "referral") = "direct";
   DTO_FIELD(Int32, intVal) = 32;
-  DTO_FIELD(List<String>, friends) = List<String>::createShared();
+  DTO_FIELD(List<String>, friends) = [](){
```

Comment:
Tip:
You can write simply like this:

```cpp
DTO_FIELD(List<String>, friends) = {"me", "he", "she" }
```

Reply:
you can find more details [here](https://github.com/oatpp/oatpp/blob/master/changelog/1.1.0.md#object-mapping-std-collections)